### DISCUSSION TITLE
1주차 회고

### DISCUSSION THREAD

**Discussion by iamyanghee** [Melon-Shake/API#31] [team]
@mews2000 @json9101 @bksooon @Lucete28 @jungssg @iamyanghee 
각자의 솔직한 생각을 아래에 코멘트 달아주셈

**Comment by iamyanghee** [comment]
@bksooon 

- 좋았던 점 - 기획 회의가 활발해서 좋았다.
- 아쉬운 점 - 좀 더 진도가 속도가 나왔으면 좋겠다
- 개선할 점 - 문서화

**Reply by iamyanghee** [reply]
캘린더나 일정 보는 사람 
나랑 당신 밖에 없다는 얘기

**Reply by Lucete28** [reply]
저도 슬쩍슬쩍 봅니다

**Comment by iamyanghee** [comment]
@mews2000 

- 좋았던 점
  - 부담이 적다<br>
- 아쉬운 점
  - 스스로 time limit을 걸 기가 힘든데 서로 걸지도 않는다.<br>
- 개선할 점
  - 자기 일정에 대해 시간을 정해놓을 수 있을지 고민이 필요

**Reply by iamyanghee** [reply]
우리 팀은 각자가 기능을 나눠서 하고 있기에
속도가 타 팀에 비해 느려보일 수 있는 지점

이라고 대신 적어봄

**Reply by Lucete28** [reply]
각자 기능을 맡아서 하는 만큼 스스로 최선을 다해서 시간을 줄이는게 중요할거 같네요

**Comment by iamyanghee** [comment]
@json9101 

- 좋았던 점
  - 서로 잘 아는 사이라 편안하고 즐거운 환경에서 프로젝트를 한다
  - 프로젝트에 의욕이 있어 아이디어가 많이 나온다
- 아쉬운 점
  - 약간 프로젝트 속도가 더딘 느낌(?) 
- 개선할 점
  - 소통이 약간 부족한 느낌이라

**Reply by bksooon** [reply]
화이팅......

**Reply by iamyanghee** [reply]
반성합니다

**Comment by iamyanghee** [comment]
@Lucete28 

- 좋았던 점 : 
    1. 시작점인데 머릿속의 이미지를 공유하고 출발할 수 있어서 좋아요
    2. 어찌저찌 진도가 나아가고 있어서 안심이 됩니다.
- 아쉬운 점 : 
    1. 과자가 자꾸 손에 묻어요
    2. 팀장님이 여론조작해요
- 개선할 점 : 앞으로 기능 구현 진도가 빨라졌으면 좋겠습니다. (그렇게 될거 같긴한데)

**Reply by bksooon** [reply]
죄송합니다....

**Comment by iamyanghee** [comment]
@jungssg 

- 좋았던 점 : 
    - 모두의 의견이 들어간 프로젝트 진행이어서 시작이 좋았다.
    - 의견을 자유롭게 나눌수 있어서 좋다. 
    - 무난한 시작이었던거 같다.
- 아쉬운 점 : 
    - 깃헙 issue, pr 작성하는게 익숙하지 않았다.
    - 테이블 구조 의견을 말할때 같은말을 여러번 전파를 해줘야해서 오래걸린 느낌
- 개선할 점 : 
    - 테이블 구조가 완료돼서 이제 속도가 붙을거 같다.

**Reply by iamyanghee** [reply]
반성

**Comment by iamyanghee** [comment]
@iamyanghee 

- 좋았던 점
  - 다들 지난 몇개월 동안 짝궁 내지는 팀플 했던 사이라서 그런지, 이미 서로 스타일을 알아서 소통하는 데 있어서 불편한 점 없이 편했다
  - 이건 아니잖아~~ 아니이~~ 오오~ 라는 식으로 편하게 뭐라 아쉬운 거나 좋은 거 바로바로 말을 꺼내서 생각을 확인하기 좋은듯
  - 그래서 팀 분위기가 좋은 듯
- 아쉬운 점
  - 근데 서로에게 긴장감을 가져야하나 싶었음
  - 아쉬운 거 얘기하면 그거에 대한 답도 편하게 변명(?) 하고 메꾸는 것 같음
  - 나도 기록 잘 못하고 누락해서 이런 말할 입장 되나 싶긴 한데, 주 초에 나만 기록해주세요 하는 것 같아서, 서운했따
- 개선할 점
  - 기록에 대한 필요성이나 기록을 하더라도 '잘' 해야하는 필요성을 모두가 가졌으면 좋겠다

**Reply by Lucete28** [reply]
[ 팀장님에 의해 지워진 댓글입니다 ]
